scio-avro/src/main/scala/com/spotify/scio/avro/nio/AvroIO.scala

```diff
@@ -90,7 +90,7 @@ case class ObjectFile[T: ClassTag](path: String)
             private[scio] def processElement(c: DoFn[T, GenericRecord]#ProcessContext): Unit =
               c.output(AvroBytesUtil.encode(elemCoder, c.element()))
           })
-          .write(AvroFile(path, AvroBytesUtil.schema))(parameters)
+          .write(AvroFile[GenericRecord](path, AvroBytesUtil.schema))(parameters)
```

Comment:
Sorry just got around to look at these PRs. Should this and the BigQuery change go to different commits since they're fixing issues with previous commits?